skeleton/Gemfile.lock

```diff
@@ -1,45 +1,71 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (2.3.5)
+    CFPropertyList (2.2.8)
```

Comment:
@jeannegreulich No idea, it's what I got when I did a `bundle update`. Maybe they retracted the newer version or something.